<?xml version="1.0" ?>
<sdf version="1.4">
  <model name="go_board">
    <static>0</static>
    <link name="link">

      <inertial>
        <mass>0.05</mass>
        <inertia>
          <ixx>0.0015</ixx>
          <ixy>0.1</ixy>
          <ixz>0.1</ixz>
          <iyy>0.0015</iyy>
          <iyz>0.1</iyz>
          <izz>0.00108</izz>
        </inertia>
      </inertial>

      <collision name="colision">
        <pose>0 0 0 0 0 0</pose>
          <geometry>
            <box>
              <size>0.42 0.42 0.02</size>
            </box>
          </geometry>
        <surface>
          <friction>
            <ode>
              <mu>30000000.0</mu>
              <mu2>3000000.0</mu2>
              <fdir1>0 0 0</fdir1>
            </ode>
          </friction>
          <contact>
            <ode>
              <kp>1000000000.0</kp>
              <kd>1000000.0</kd>
              <min_depth>0.0025</min_depth>
              <max_vel>0.1</max_vel>
            </ode>
          </contact>
        </surface>
      </collision>

      <visual name="visual1">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
             <uri>model://go_board/meshes/go_board.dae</uri>
    		<!-- the dae is a cube of 5x5x5 cm and the chessboard is 46x46x4-->
	    <scale> 8.4 8.4 0.4 </scale>
          </mesh>
        </geometry>
      </visual>

      <velocity_decay>
        <linear>0.1</linear>
        <angular>0.1</angular>
      </velocity_decay>
      <self_collide>0</self_collide>
      <kinematic>0</kinematic>
      <gravity>1</gravity>
    </link>
  </model>
</sdf>

